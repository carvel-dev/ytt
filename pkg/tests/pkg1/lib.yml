#@ load("@ytt:lib", "lib")
---
lib1:
  hello: #@ lib.eval("@lib1").text("hello.yml")
  all: #@ lib.eval("@lib1").text()
  foo: #@ lib.eval("@lib1", {"arg2": "arg2val"}).text("foo.yml")
lib2:
  all: #@ lib.eval("lib2", {"arg": "argval"}).text()
