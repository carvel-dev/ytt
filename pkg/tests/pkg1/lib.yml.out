lib1:
  hello: |
    hello: world
  all: |
    foo: bar
    argval: null
    arg2val: default
    ---
    hello: world
  foo: |
    foo: bar
    argval: null
    arg2val: arg2val
lib2:
  all: |
    where: lib2
    argval: argval
    arg2val: default
